source/index.html

```diff
@@ -4,9 +4,169 @@
+    Занялся собой? Займись котом!
+  </p>
+  <img src="#" alt="Кэт Энерджи с курицей." class="product__image">
+  <button class="product__button button">
+    Подобрать программу
+  </button>
+</section>
+<section class="main-product-list">
+  <h2 class="visually-hidden">
+    Варианты программ.
+  </h2>
+  <ul class="product-list">
+    <li class="product-list__item product-list__item--slim">
+      <h2 class="product-list__title">
+        Похудение
+      </h2>
+      <p class="product-list__subtitle">
+        Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.
+      </p>
+<a href="#" class="product-list__link product-list__link-arrow">
+  Каталог SLIM
+ </a>
+    </li>
+    <li class="product-list__item product-list__item--pro">
+    <h2 class="product-list__title">
+ Набор массы
+      </h2>
+      <p class="product-list__subtitle">
+        Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!
+      </p>
+<a href="#" class="product-list__link product-list__link-arrow">
+  Каталог PRO
+</a>
+    </li>
+  </ul>
+</section>
+<section class="advantages">
+  <h2 class="advantages__header">
+    Как это работает
+  </h2>
+  <ul class="advantages-list">
+    <li class="advantages-list__item advantages__item--eco">
+      Функциональное питание
+      содержит только полезные
+      питательные вещества.
+    </li>
+    <li class="advantages-list__item advantages__item--easy">
+      Выпускается в виде порошка, 
+который нужно лишь залить 
+кипятком и готово.
+    </li>
+    <li class="advantages-list__item advantages__item--economy">
+      Замените один-два приема 
+обычной еды на наше 
+функциональное питание.
+    </li>
+    <li class="advantages-list__item advantages__item--fast">
+      Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+    </li>
+  </ul>
+</section>
+<section class="example">
+  <h2 class="example__title">
+Живой пример
+  </h2>
+  <p class="example__subtitle">
+    Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. 
+  </p>
+  <p class="example__stats">
+   5 КГ
+  </p>
+  <p class="example__description">
+снижение веса
+  </p>
+  <p class="example__stats">
+ 60 ДНЕЙ
+   </p>
+   <p class="example__description">
+затрачено времени
+   </p>
+   <p class="example__item">
+    ЗАТРАТЫ НА ПИТАНИЕ: 15 000 РУБ.
+   </p>
+   <img src="#" alt="Фото кота до программы питания." class="example__image--fat">
+   <img src="#" alt="Фото кота после программы питания." class="example__image--slim">
+   <button class="example__image-button">
+   </button>
+</section>
+</main>
+   <footer class="footer">
+   <div class="footer__contacts contacts">
+    <h2 class="footer__title">
```

Comment:
```suggestion
    <h2 class="contacts__title">
```
зачем тогда было делать выше див блок+элемент? footer__contacts contacts
загооловк и адрес относятся к блоку contacts